samples/caf_sensor_manager/sysbuild.cmake

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if("${SB_CONFIG_REMOTE_BOARD}" STREQUAL "")
+  return()
```

Comment:
message FATAL_ERROR

Reply:
That part is intentional - not all configurations have remote board